### ISSUE TITLE
'Unknown function' Errors when loading patch in pd-extended

### ISSUE THREAD

**Issue by andrew-e** [megalon/pd-AUTOMATONISM-sampler#1]
After following the instructions in the README, I load up the module from the modules list.  After I do that I get these following errors:

```
** ignoring width or font settings from future Pd version **
list fromsymbol: unknown function
 list fromsymbol
... couldn't create
list tosymbol: unknown function
 list tosymbol
... couldn't create
list fromsymbol: unknown function
 list fromsymbol
... couldn't create
list tosymbol: unknown function
 list tosymbol
... couldn't create
list fromsymbol: unknown function
 list fromsymbol
... couldn't create
list tosymbol: unknown function
 list tosymbol
... couldn't create
 array define $0-sample-L
... couldn't create
 array define $0-sample-R
... couldn't create
$1: argument number out of range
statesave/5-sampler-ss.txt: can't open
statesave/5-sampler-ss.txt: read failed
```

I am running this on pd-extended on a mac.  Both `tosymbol` and `fromsymbol` are objets that i can create, so I'm not sure why it is having issues with this.

I will see if the issue continues on vanilla pd.

**Comment by andrew-e** [comment]
Yeah, just tried with vanilla pd, no issue.  So this is a pd-extended specific problem.

**Comment by megalon** [comment]
I only use these objects to display the filename of the sample on the GUI. I just tried building a version for extended, and I can't figure out how to split a symbol apart using an arbitrary character as a the split point. I've uploaded a new file to the repo called "sampler-for-extended.pd", which is a version of the sampler that works with pd-extended, but it uses a symbol box to display the full path to the sample file, so it doesn't look as nice.

This is in fact a problem with pd-extended. "list tosymbol" and "list fromsymbol" are not available in the version of PD that extended was based on. There are the similar objects "tosymbol" and "fromsymbol" that are part of the cyclone library, but they don't actually do the same thing.

The vanilla [list tosymbol] converts a list of numbers to the ascii characters associated with those numbers, and [list fromsymbol] does the opposite.

cyclone's [tosymbol] converts a list directly into a single symbol, and [fromsymbol] splits a symbol into a list.
